packages/driver/src/multi-domain/events/misc.ts

```diff
@@ -14,12 +14,6 @@ export const handleMiscEvents = (Cypress: Cypress.Cypress, cy: $Cy) => {
     cy.state(state)
   })
 
-  // Listen for window load events from the primary window to resolve page loads
-  Cypress.specBridgeCommunicator.on('window:load', ({ url }) => {
```

Comment:
This was removed to be added conditionally in the `multi-domain/cypress.ts` file.